ui/client/http/HttpService.ts

```diff
@@ -258,11 +258,12 @@ class HttpService {
         .map(({performedAt}) => performedAt))
   }
 
-  deploy(processId, comment?) {
+  deploy(processId, comment?): Promise<{isSuccess: boolean}> {
     return api.post(`/processManagement/deploy/${encodeURIComponent(processId)}`, comment).then(() => {
       this.addInfo(i18next.t("notification.info.scenarioDeployed", "Scenario {{processId}} was deployed", {processId}))
       return {isSuccess: true}
     }).catch(error => {
+      if (error?.response?.status == 400) throw error
```

Comment:
I don't understand that.
1. when this error is thrown and when this piece of code below is performed?
2. what will happen when this error would be thrown? "broken nut" will appear?

Reply:
ad1 changed it so it looks more clear now, pop up shows if it's not validation error, else i rethrow error
ad2 no such extremes, it's caught immediately above :)